src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsTextHelper.cs

```diff
@@ -106,12 +106,16 @@ private static string HumanizeMultilineCodeTags(this string text)
 
         private static string HumanizeParaTags(this string text)
         {
-            return ParaTag().Replace(text, (match) => "<br>" + match.Groups["display"].Value);
+            return ParaTag().Replace(text, match =>
+            {
+                var paraText = "<br>" + match.Groups["display"].Value.Trim();
+                return LineBreaks().Replace(paraText, _ => string.Empty);
+            });
         }
 
         private static string HumanizeBrTags(this string text)
         {
-            return BrTag().Replace(text, m => Environment.NewLine);
+            return BrTag().Replace(text, _ => Environment.NewLine);
```

Comment:
it seems complicated to make tests pass on macos and ubuntu because for unix `\n` is used for line breaks and here we replacing `<br>` tag as `Environment.NewLine`. But in `...v1.verified.txt` files you're using specific line breaks `\r\n`.

I've searched the codebase and didn't find any examples of `<para>` in /test/WebSites

Reply:
Maybe there's a setting in [Verify](https://github.com/VerifyTests/Verify) that can be used to ignore line endings in specific tests?

Reply:
good idea, let me try to dig something

Reply:
I couldn't find anything useful in `Verify` link. I also tried more elegant solution with `.ReplaceLineEndings()` and regex expressions but no avail